packages/react/src/index.tsx

```diff
@@ -33,3 +33,4 @@ export type {
   FiatAmount,
   Matic,
 } from '@lens-protocol/shared-kernel';
+export { CustomFiltersTypes } from '@lens-protocol/api-bindings';
```

Comment:
Does this belongs here? It's used in both publication and profile so it doesn't make sense to be inside one of those folders.